scalafix-core/shared/src/main/scala/scalafix/internal/config/DisableUnlessConfig.scala

```diff
@@ -3,52 +3,57 @@ package scalafix.internal.config
 import metaconfig.{Conf, ConfDecoder, ConfError, Configured}
 import org.langmeta.Symbol
 
+import scalafix.CustomMessage
 import scalafix.internal.config.MetaconfigPendingUpstream.XtensionConfScalafix
 import scalafix.internal.util._
 
 case class UnlessConfig(
-    block: Symbol.Global,
-    symbol: Symbol.Global,
-    message: Option[String])
+    unless: Symbol.Global,
+    symbols: List[CustomMessage[Symbol.Global]])
 
 object UnlessConfig {
-  implicit val decoder: ConfDecoder[UnlessConfig] =
+  implicit val customMessageReader: ConfDecoder[CustomMessage[Symbol.Global]] =
+    CustomMessage.decoder(field = "symbol")
+
+  implicit val reader: ConfDecoder[UnlessConfig] =
     ConfDecoder.instanceF[UnlessConfig] {
       case c: Conf.Obj =>
-        (c.get[Symbol.Global]("block") |@|
-          c.get[Symbol.Global]("symbol") |@|
-          c.getOption[String]("message")).map {
-          case ((a, b), c) => UnlessConfig(a, b, c)
+        (c.get[Symbol.Global]("unless") |@|
```

Comment:
Pro tip, with `getField` you don't have to specify the type parameter. But doesn't matter much anyways, very soon we can use generic derivation to avoid this boilerplate in the first place.